src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -91,64 +81,22 @@ pub(crate) struct SharedBufferBatchInner {
 
 impl SharedBufferBatchInner {
     pub(crate) fn new(
-        table_id: TableId,
         epoch: HummockEpoch,
         spill_offset: u16,
         payload: Vec<SharedBufferItem>,
-        delete_ranges: Vec<(Bound<Bytes>, Bound<Bytes>)>,
         size: usize,
         _tracker: Option<MemoryTracker>,
     ) -> Self {
-        let point_range_pairs = delete_ranges
-            .into_iter()
-            .map(|(left_bound, right_bound)| {
-                (
-                    match left_bound {
-                        Bound::Excluded(x) => PointRange::from_user_key(
-                            UserKey::new(table_id, TableKey(x.to_vec())),
-                            true,
-                        ),
-                        Bound::Included(x) => PointRange::from_user_key(
-                            UserKey::new(table_id, TableKey(x.to_vec())),
-                            false,
-                        ),
-                        Bound::Unbounded => unreachable!(),
-                    },
-                    match right_bound {
-                        Bound::Excluded(x) => PointRange::from_user_key(
-                            UserKey::new(table_id, TableKey(x.to_vec())),
-                            false,
-                        ),
-                        Bound::Included(x) => PointRange::from_user_key(
-                            UserKey::new(table_id, TableKey(x.to_vec())),
-                            true,
-                        ),
-                        Bound::Unbounded => PointRange::from_user_key(
-                            UserKey::new(
-                                TableId::new(table_id.table_id() + 1),
-                                TableKey::default(),
-                            ),
-                            false,
-                        ),
-                    },
-                )
-            })
-            .collect_vec();
-
-        let SharedBufferDeleteRangeMeta {
-            smallest_empty,
-            mut smallest_table_key,
-            mut largest_table_key,
-            point_range_pairs,
-        } = Self::get_table_key_ends(table_id, point_range_pairs);
+        let mut largest_table_key = Bound::Included(Bytes::new());
```

Comment:
In the removed method `get_table_key_ends`, we have the following initial value. When there is no delete range, the value should remain unchanged.
```
let mut largest_table_key = Bound::Included(Bytes::new());
let mut smallest_table_key = BytesMut::new();
let mut smallest_empty = true;
```